test/linters_test.go

```diff
@@ -183,10 +183,9 @@ func extractRunContextFromComments(t *testing.T, sourcePath string) *runContext
 	scanner := bufio.NewScanner(f)
 	for scanner.Scan() {
 		line := scanner.Text()
-		if !strings.HasPrefix(line, "//") {
+		if strings.HasPrefix(strings.TrimSpace(line), "package") {
```

Comment:
Here I did a not typical change for linter contributing:
Before this change test parameters can be only in first comment lines.
After this change test parameters can be set in any comments line before key work `packgage`

Reply:
This needed to add possible to test go-header with golangci testing approach.